### PR TITLE
P#77276 Add onOffice shortcuts to the admin bar

### PR DESCRIPTION
**Pull request by LongTrong-exe**
Related to issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/311

changed logs:

### CODE REVIEW

File: plugin.php [hunk 1]
```diff
@@ -265,4 +264,65 @@ function update_duplicate_check_warning_option()
...
+		exit();
+	}
+});
+
+if ( ! is_admin() ) {
+	add_action('admin_bar_menu', function ( $wp_admin_bar ) {
+		$toolBarConfig = [
+			[
+				'id'    => 'onoffice',
+				'title' => 'onOffice',
```
Comment (jmaas-onoffice) [inline plugin.php:281]: I think the titles need to be translated, right?

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]

File: plugin.php (hunk 1 above)
Comment (dai-eastgate) [reply plugin.php]: I added the translation, please check the PR for me!

Comment (LongTrong-exe) [reply plugin.php]: I added the translation, please check the PR for me!

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]
This was missing in the requirements, but the menu should only be shown for editors and admins.

**Comment by LongTrong-exe** [comment]
@jmaas-onoffice  I updated, could you review?

**Review by jmaas-onoffice — APPROVED** [review]

**Pull request merged**